incubator/virtualcluster/example/cr-syncer/api/cr-project/clientset/v1alpha1/api.go

```diff
@@ -0,0 +1,39 @@
+package v1alpha1
```

Comment:
So that we don't have to copy over types and clients as well as have references to non-k8s owned projects we could we use the crds/crs from https://github.com/kubernetes/sample-controller/ specifically `Foos`, then we can just use those by reference and not need to maintain the generated code.